PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -1037,6 +1047,19 @@ void PATMuonProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
       }
     }
 
+    // MVA ID
+    float mvaID = 0.0;
+    if (computeMuonIDMVA_) {
+      if (muon.isLooseMuon() && muon.passed(reco::Muon::PFIsoTight)) {
+        mvaID = globalCache()->muonMvaIDEstimator().computeMVAID(muon)[1];
+      } else {
+        mvaID = -99;
```

Comment:
these magic numbers should be put in a named constexpr

Reply:
Do you mean to define '-99' like (for example): 'constexpr int MVAsentinelValue=-99'.

Reply:
right, for -99. I think the working points 0.12, 0.48 should actually be configurable parameters of the producer.